spec/models/amendments/modification_spec.rb

```diff
@@ -27,6 +27,18 @@
     it { is_expected.to eq(1.0) }
   end
 
+  describe '#locked_section' do
+    subject { modification.valid? }
```

Comment:
I woud keep the modification as subject and then I would check it like this:

```
it { is_expected.not_to be_locked_section }
it { is_expected.to be_valid }
```
after that, I would test different use cases to test that locked_section is ok.

Reply:
> I woud keep the modification as subject and then I would check it like this:
> 
> ```
> it { is_expected.not_to be_locked_section }
> it { is_expected.to be_valid }
> ```
> 
> after that, I would test different use cases to test that locked_section is ok.

🤔 Like this?  https://github.com/podemos-info/national_budgets/pull/44/commits/9bc9c07e0b6b2cd1ff03514c88c8fd582202759e

Reply:
Or like this?
https://github.com/podemos-info/national_budgets/pull/44/commits/ed11fdfc45f96c147b150ec1c3b9b31a7305e358